docs/reference/child_admin.rst

```diff
@@ -18,14 +18,21 @@ its parent:
 
 .. configuration-block::
 
+    .. code-block:: yaml
+
+        # app/config/services.yml
+        App\Admin\PlaylistAdmin:
+            calls:
+                - [ addChild, ['@App\Admin\VideoAdmin', 'playlist']]
+
     .. code-block:: xml
 
-        <!-- app/config/config.xml -->
-        <service id="sonata.admin.playlist" class="App\Admin\PlaylistAdmin">
+        <!-- app/config/services.xml -->
+        <service id="App\Admin\PlaylistAdmin" class="App\Admin\PlaylistAdmin">
```

Comment:
So we still need the class attribute?

Reply:
not sure for XML 🤔

Reply:
Please make sure.

Reply:
we keep them in all our config files, so the docs should be the same.

Reply:
@OskarStark [here](https://symfony.com/doc/current/service_container.html#service-parameters) such services are described with `id` attribute only (see XML tab)

Reply:
@OskarStark looks like we made a mistake then :sweat_smile: please remove them